header/index.html

```diff
@@ -145,6 +145,7 @@ <h2>With navigation links</h2>
                     class="form-control"
                     rows="5"
                     required="required"
+                    aria-required="true"
```

Comment:
Isn't aria-required superfluous  when `required` is specified?

Reply:
I was under the understanding you might need both at least for older browsers but maybe it is moot. https://usability.yale.edu/web-accessibility/articles/forms

Reply:
"Older versions of HTML" is different from "older browsers" though – all our stuff is HTML5, so the `required` attribute should always do the same thing as `aria-required`, I think